ocu/open_event.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import re
+import subprocess
+import sys
+from typing import Optional
+
+from ocu.prefs import PrefName, prefs
+
+
+def should_open_google_meet_app(url: Optional[str]) -> bool:
+    """
+    Return True if the given URL is a Google Meet URL, and the user prefers to open it in the native app;
+    return False otherwise
+    """
+    if not url:
+        return False
+
+    matches = re.search(r"https://(meet\.google\.com)", url)
+    is_google_url = bool(matches)
+    if not is_google_url:
+        return False
```

Comment:
@Avivbens I think this can be streamlined slightly:

```suggestion
    is_google_url = bool(re.search(r"https://(meet\.google\.com)", url))
    if not is_google_url:
        return False
```